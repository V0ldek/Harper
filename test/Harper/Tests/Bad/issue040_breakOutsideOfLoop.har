main :: sideeffect -> ();
main = {
  var (i :: Integer) = 0;
  while i < 10 {
    eval printLn i ();
    i += 1;
  }
  break;
};