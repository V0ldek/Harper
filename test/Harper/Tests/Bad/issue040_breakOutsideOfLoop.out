Error: a `break` statement cannot be used outside of a loop body.
During evaluation of:
 break ;

Located at line 8 column 3

Execution terminated with an error: type error.